rasa/core/training/story_reader/markdown_story_reader.py

```diff
@@ -29,11 +29,14 @@ class MarkdownStoryReader(StoryReader):
 
     """
 
-    async def read_from_file(self, filename: Text,) -> List[StoryStep]:
+    async def read_from_file(self, filename: Text) -> List[StoryStep]:
         """Given a md file reads the contained stories."""
 
         try:
-            with open(filename, "r", encoding=io_utils.DEFAULT_ENCODING) as f:
+            # TODO: Hack by Genie for Windows
+            with open(
+                filename, "r", encoding=io_utils.DEFAULT_ENCODING, errors="ignore"
+            ) as f:
```

Comment:
is it a general solution? current master doesn't work on Windows?

Reply:
Tbh, I assume it is my laptop being stupid about things... I'm sure if it didn;t work people would complain a mln times!